src/current/advisories/a104309.md

```diff
@@ -80,6 +80,32 @@ If data is found, it can be re-emitted in two ways:
 - By restarting the changefeed with `initial_scan` set to `yes` and without a cursor timestamp. This will perform a full export of all current data across the changefeed.
 - By restarting the changefeed and providing a cursor timestamp. If a cursor timestamp is provided, an initial scan will not be performed. Instead, updates above the cursor timestamp will be emitted. The cursor timestamp can be initialized below the timestamp of omitted writes (refer to the reference documentation for [CREATE CHANGEFEED](https://www.cockroachlabs.com/docs/stable/create-changefeed)). However, specifying a cursor timestamp farther in the past than the [`gc.ttlseconds`](https://www.cockroachlabs.com/docs/stable/configure-replication-zones#gc-ttlseconds) replication zone variable, which defaults to `14400` seconds (4 hours), results in an error.
```

Comment:
```suggestion
- By restarting the changefeed and providing a cursor timestamp. If a cursor timestamp is provided, an initial scan will not be performed. Instead, updates above the cursor timestamp will be emitted. The cursor timestamp can be initialized below the timestamp of omitted writes (refer to the reference documentation for [`CREATE CHANGEFEED`](https://www.cockroachlabs.com/docs/stable/create-changefeed)). However, specifying a cursor timestamp farther in the past than the [`gc.ttlseconds`](https://www.cockroachlabs.com/docs/stable/configure-replication-zones#gc-ttlseconds) replication zone variable, which defaults to `14400` seconds (4 hours), results in an error.
```

Reply:
There is some formatting issue earlier on this page with:

> '{timestamp}with the timestamp expressed in nanoseconds with the logical component after the decimal point. For example, to query for the logged timestamp1707854079.38430848,2`

It isn't rendering because of the missing backtick.

Reply:
@mdlinville This, under **Mitigation** is not rendering:

> '{timestamp}with the timestamp expressed in nanoseconds with the logical component after the decimal point. For example, to query for the logged timestamp1707854079.38430848,2`

Reply:
Wow, it was  single quote instead of a backtick! Fixed.